index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Intuit Sign in page</title>
+</head>
+<body>
+    <main>
+        <div id="up-info">
+            <p>
+                Already have an Intuit account? <br>
+                <a href="#">I forgot my User ID or Password</a>
+            </p>
+            <button class="top-button"><i class="fa fa-lock" aria-hidden="true">&nbsp; Sign in</i></button>
+        </div>
+        <section>
+            <div>
+                <img src="img/LogosP.png" alt="Logos">
+            </div>
+            <div class="central-section">
+                <div id="form">
+                    <p >
+                        <h3>Create an Intuit account</h3><br>
+                    </p>
+                    <p>
+                        One account for everything Intuit, including Mint.
+                        <br> <a href="#">Learn more</a>
+                    </p>
+                    <form action="index.html">
+                        <fieldset class="form-info">
+                            <label for="email"><p>Email address</p></label>
+                            <input name="email" type="email">
+                            <label for="number"><p>Phone (recomended)</p></label>
+                            <input name="number" type="text">
```

Comment:
- [ ] This input section should have a type of "tel". You can take a look at [this article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) for more information on Form input types. 

![Screen Shot 2020-07-15 at 12 52 16 AM](https://user-images.githubusercontent.com/48804258/87484014-6fd4ba00-c635-11ea-92df-e019eb056fa6.png)